hellman_example/sort_on_disk.hpp

```diff
@@ -145,7 +145,7 @@ class FileDisk : public Disk {
         f_.rdbuf()->pubsetbuf(buf_, BUF_SIZE);
 
         if (!f_.is_open()) {
-            std::cout << "Fialed to open" << std::endl;
```

Comment:
this change appears to be conflicting with something happening in `main`. The files under `hellman_example` are also not actually used. afaik, they're left as a historical curiosity.

Reply:
The line below it changed from a `string` to a `runtime_error`, no direct conflict. Resolved and pushed.